1/assignment-week1.py

```diff
@@ -0,0 +1,83 @@
+print ("Week 1 Assignment - Math Equation Evaluator")
+print ("Intern - Shu Wei")
+print ("")
+
+eArray = list(map(str, input("Please enter your input with spaces between them: ").split()))
+
+#Definitions
+aCount = (len(eArray))
+fCount = 3
+nCount = 4

```

Comment:
python variable names are usually snake cases

snake_case_variable
camelCaseVariable
PascalCaseVariable

iDioTcAsEVariabLE

Reply:
Question. If a variable is being called in a very frequent basis, wouldn't it be bad to have a long name since you'll need to type that long name every time?